EIPS/eip-1620.md

```diff
@@ -1,296 +1,185 @@
 ---
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg)
+author: Paul Razvan Berg (@PaulRBerg)
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Last Call
 type: Standards Track
 category: ERC
 created: 2018-11-24
+requires: 20
+review-period-end: 2020-01-11
```

Comment:
If you are still aiming to last call this, let's merge it, but please change the `review-period-end` to be valid, e.g at least 14 days from today.

Reply:
Can we make it "Final"?

Reply:
Yes, after it has passed the actual last call time in the RSS feed. Blame the editors for getting this merged at the time it should have been :(

Reply:
Sorted. Happy to wait.